documentation/prototype-usage-instructions.md

```diff
@@ -63,51 +98,322 @@ context of the document. This can be done with additional keys that
 have been added to the `\includegraphics` command:
 
 ```latex
-\includegraphics[alt={this shows a yellow duck}]{duckimage}
-\includegraphics[artifact]{dekoration}
+\includegraphics[height=4cm,alt={Portrait of Shakespeare}]{william-shakespeare.jpg}
+\includegraphics[height=4cm,artifact]{crinklepaper}\makebox[0pt][r]{Some text }
 ```
 
 ## Handling tables (`tabular`) in the document
 
 If the document contains data tables, the author has to identify the
 header rows of the tables so that they can be tagged as `<TH>`
 cells. By default all cells are considered data cells. This can be done
+generally (in the preamble) or on a table by table basis before the table.
+as shown in the following example.
 ```latex
 \tagpdfsetup{table/header-rows={1,2}}
+\begin{tabular}{lr}
+\multicolumn{2}{c}{Example}\\
+Name&Value\\
+This& 11 \\
+That & 2
+\end{tabular}
 ```
-generally (in the preamble) or on a table by table basis before the table.
+
 
 If a table should not be tagged as table, for example, because it is
 merely used to ensure that the content is properly aligned, it should
-be turned into a presentation table with
+be turned into a presentation table with the `table/tagging=presentation`
+key as shown below.
 ```latex
 \tagpdfsetup{table/tagging=presentation}
+\begin{tabular}{ccc}
+\textbullet & \textbullet & \textbullet \\
+--- & --- & ---
+\end{tabular}
 ```
```

Comment:
maybe  something like 

At the moment this isn't transfered well by the derivation algorithm from PDF to html, but this area will see improvements (the PDF structe is already correct, as one cane see in ngPDF).

Reply:
5be7bef